src/components/Main.jsx

```diff
@@ -2,8 +2,11 @@ import styles from '../styles/main'
 
 import React from 'react'
 
+import ButtonClientMobile from './ButtonClientMobile'
+
 const Main = ({ children }) => (
   <main class={styles['fil-content']}>
+    { __TARGET__ !== 'mobile' && <ButtonClientMobile />}
```

Comment:
You need to declare the `__TARGET__` variable as a global variable for the lint analysis:

Put on first line
```
/* global __TARGET__ */
```